testdata/src/generated_included/main.go

```diff
@@ -0,0 +1,41 @@
+// Code generated by "stringer -type=SomeType"; DO NOT EDIT.
```

Comment:
Maybe replace:

```
// Code generated by "stringer -type=SomeType"; DO NOT EDIT.
```

with

```
// Code generated by "example_generator"; DO NOT EDIT.
```

or something like this? Because stringer looks a bit misleading here.